components/ModSubmissionLeftNavbar.vue

```diff
@@ -0,0 +1,154 @@
+            >
+                {{ $t("modPanelFacilitySubmissionLeftNavbar.addresses") }}
+            </button>
+            <button
+                data-testid="submission-form-leftnav-googleMapsInformation"
+                :class="{ 'bg-secondary': activeSection === ModSubmissionLeftNavbarSectionIDs.GoogleMapsInformation,
+                          'bg-primary-inverted': activeSection !== ModSubmissionLeftNavbarSectionIDs.GoogleMapsInformation }"
+                class="w-full py-4 my-2 text-sm text-start pl-2 rounded border-b-2 border-tertiary-bg"
+                @click="scrollToSectionOfForm(ModSubmissionLeftNavbarSectionIDs.GoogleMapsInformation)"
+            >
+                {{ $t("modPanelFacilitySubmissionLeftNavbar.googleMapsInformation") }}
+            </button>
+            <button
+                data-testid="submission-form-leftnav-healthcareProfessionalIds"
+                :class="{ 'bg-secondary': activeSection === ModSubmissionLeftNavbarSectionIDs.HealthcareProfessionalIds,
+                          'bg-primary-inverted': activeSection !== ModSubmissionLeftNavbarSectionIDs.HealthcareProfessionalIds }"
+                class="w-full py-4 my-2 text-sm text-start pl-2 rounded border-b-2 border-tertiary-bg"
+                @click="scrollToSectionOfForm(ModSubmissionLeftNavbarSectionIDs.HealthcareProfessionalIds)"
+            >
+                {{ $t("modPanelFacilitySubmissionLeftNavbar.healthcareProfessionalIds") }}
+            </button>
+            <button
+                data-testid="submission-form-leftnav-changeLog"
+                :class="{ 'bg-secondary': activeSection === ModSubmissionLeftNavbarSectionIDs.ChangeLog,
+                          'bg-primary-inverted': activeSection !== ModSubmissionLeftNavbarSectionIDs.ChangeLog }"
+                class="w-full py-4 my-2 text-sm text-start pl-2 rounded border-b-2 border-tertiary-bg"
+                @click="scrollToSectionOfForm(ModSubmissionLeftNavbarSectionIDs.ChangeLog)"
+            >
+                {{ $t("modPanelFacilitySubmissionLeftNavbar.changeLog") }}
+            </button>
+        </div>
+    </div>
+</template>
+
+<script setup lang="ts">
+import { ref, type Ref, onMounted, onUnmounted } from 'vue'
+import { useModerationSubmissionsStore } from '~/stores/moderationSubmissionsStore'
+import { ModSubmissionLeftNavbarSectionIDs } from '~/stores/moderationScreenStore'
+import { handleScroll, scrollToSectionOfForm, type SectionInformation } from '~/utils/handleScroll'
+
+const moderationSubmissionsStore = useModerationSubmissionsStore()
+const activeSection: Ref<string> = ref(ModSubmissionLeftNavbarSectionIDs.ContactInformation)
+const isScrolling: Ref<boolean> = ref(false)
+
+const observeSubmissionFormSections = () => {
+    const sections: NodeListOf<Element> = document.querySelectorAll('.facility-form-section')
+
+    const addressObserver = new IntersectionObserver(
```

Comment:
these observers still look like they could be genericized and moved to the util with the threshold values as parameters.

Reply:
I DID IT!!!! I think @ermish